Sources/Display/Platform/Win32/win32_window.cpp

```diff
@@ -1685,8 +1735,6 @@ void Win32Window::create_hid_devices()
 
 void Win32Window::get_styles_from_description(const DisplayWindowDescription &desc, DWORD &style, DWORD &ex_style)
 {
-	style = 0;
-	ex_style = 0;
```

Comment:
Yes, I see your point. It is actually confusing at the moment, due to the caller using 
		DWORD style = 0, ex_style = 0;
		get_styles_from_description(window_desc, style, ex_style);
I'll commit, and clarify the offending function